### PR TITLE
Replace AnySequence by AnyRandomAccessCollection

### PR DESCRIPTION
**Pull request by ddddxxx**

### CODE REVIEW

**Comment by harlanhaskins** [comment]
Thank you! This looks awesome!

Small comments, but I'd love to merge this.

File: Sources/Clang/Utilities.swift [hunk 1]
```diff
@@ -55,3 +55,9 @@ internal class Box<T> {
   public var value: T
   init(_ value: T) { self.value = value }
 }
+
+extension AnyRandomAccessCollection {
+  init<T: Strideable & ExpressibleByIntegerLiteral>(count: T, transform: (T) -> Element) {
```
Comment (harlanhaskins) [inline Sources/Clang/Utilities.swift]: Can you add a doc comment explaining what this is? Also, I'd rename `transform` to something like `indexingOperation`

**Comment by ddddxxx** [comment]
@harlanhaskins Done!

**Comment by harlanhaskins** [comment]
Wonderful! Thank you!

**Pull request merged**